### PR TITLE
feature: add alerts to graphs

### PR DESCRIPTION
**Pull request by technotronic12**
There is no use in having a dashboard without the ability to add alerts to it. I've added this option.

Also, I've noticed that alerts that are set on ref targets (`divideSeries(#B, #A)`) are not working. They are missing a field called `targetFull` which Grafana evaluates alerts upon. I've added this field when adding a target that contains ref metrics.

This solves https://github.com/uber/grafana-dash-gen/issues/58

### CODE REVIEW

**Comment by technotronic12** [comment]
There is some mixup with const/var that I don't like, I can probably run a quick rename to all vars if needed... All supported node versions support this syntax.

**Comment by technotronic12** [comment]
Looks like there is something else needed for the alerts to work, I will update once this one is ready.

**Comment by technotronic12** [comment]
@albertyw any chance this one is getting reviewed anytime soon or should I just use a fork?

File: grafana/panels/graph.js [hunk 1]
```diff
@@ -106,11 +107,49 @@ Graph.prototype.generate = function generate() {
...
-        target: target.toString(),
-        hide: target.hide
-    });
+    const refs = 'abcdefghijklmnopqrstuvwxyz';
+    const builtTarget = target.toString();
+
+    const targetWithRefs = {
+        target: builtTarget,
+        hide: target.hide,
+        refId: refs[this._currentRefIndex++].toUpperCase(),
```
Comment (albertyw) [inline grafana/panels/graph.js:121]: instead of `toUpperCase()`, you could have `refs = 'ABCDEFG...` above

File: grafana/alert/alert.js [hunk 2]
```diff
@@ -17,22 +15,34 @@ function Alert(opts) {
     handler: 1,
   };
 
+  this._init(opts);
+  this._initConditions(opts);
+}
+
+Alert.prototype._init = function _inti(opts) {
```
Comment (albertyw) [inline grafana/alert/alert.js:22]: typo - `init` instead of `inti`

**Pull request merged**